src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/components/base/ImageComponent.tsx

```diff
@@ -38,13 +39,11 @@ export function ImageComponent(props: IImageProps) {
     useSelector((state: IRuntimeState) => state.profile.profile.profileSettingPreference.language);
   const width = props.image.width || '100%';
   const align = props.image.align || 'center';
-  const altText = props.getTextResourceAsString(props.textResourceBindings.altTextImg) || 'image';
+  const altText = props.getTextResourceAsString(props.textResourceBindings.altTextImg);
 
-  let imgSrc = props.image.src.nb;
-  if (language === 'en' && props.image.src.en) {
-    imgSrc = props.image.src.en;
-  } else if (language === 'nn' && props.image.src.nn) {
-    imgSrc = props.image.src.nn;
+  let imgSrc = props.image.src[language] || props.image.src[Object.keys(props.image.src)[0]];
+  if (imgSrc.startsWith('wwwroot')) {
+    imgSrc = imgSrc.replace('wwwroot', `/${(window as Window as IAltinnWindow).org}/${(window as Window as IAltinnWindow).app}`);
```

Comment:
Not sure I get what the advantage of this is? It adds more complexity to the code, without giving much value it seems - but maybe I'm missing something :)

Reply:
It's hugely valuable when renaming apps to host them on a different name (fewer files to change). It is also slightly valuable for my altinn2converter, as I don't need to know the app name, when generating, and the same for most users.

Reply:
I see, thanks for the explanation. I think it makes sense to include something like this. Is `wwwroot` the correct/most common term? What about `public`?  

This should perhaps be included in the documentation on https://docs.altinn.studio/app/development/ux/images/#add-images-to-the-application.

Reply:
Please see my attached PR :) https://github.com/Altinn/altinn-studio-docs/pull/352

Reply:
`public` or `appfiles` are certantly good alternative names. Not sure how you usually settle naming discussions?

Reply:
I'll raise this internally, and post an update here later today :)

Reply:
I just learned that `wwwroot` is the default name used for static files in ASP.NET Core (https://docs.microsoft.com/en-us/aspnet/core/fundamentals/static-files?view=aspnetcore-6.0#serve-static-files). So makes sense to keep that name :)

Reply:
Yes. I picked that name because it is a reference to files that is in a folder called `wwwroot`. It is a permanent name (hard to change after apps starts using it), so some discussion on naming is to be expected.